src/openforms/formio/rendering/nodes.py

```diff
@@ -36,14 +36,16 @@ class ComponentNode(Node):
     step: SubmissionStep
     depth: int = 0
     is_layout = False
-    path: Path = None
+    path: Path = None  # Path in the data (#TODO rename to data_path?)
+    configuration_path: Path = None  # Path in the configuration tree
```

Comment:
when/if we rename it, we should also take `openforms.formio.utils.flatten_by_path` into account, that's yet another "configuration" path (it's the json path that you can feed to glom given a formio configuration, e.g. `components.0.columns.3` and gives you the component at that location)